Week03/lesson06/homework/startup_matchmaker/css/developersstyle.css

```diff
@@ -0,0 +1,187 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+html {
+  background-color: #bebebe;
+}
+
+body {
+  line-height: 1.5;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+header {
+  height: 100px;
+  background-color: #000;
+  border-bottom: 6px solid yellow;
+  margin-bottom: 15px;
+}
+
+.logo {
+  display: inline-block;
+  padding-top: 25px;
+}
+
+.logo a {
+  font-size: 30px;
+  text-decoration: none;
+  color: #fff;
+}
+
+nav a {
+  display:inline-block;
+  height: 94px;
+  padding-top: 35px;
+  padding-left: 10px;
+  padding-right: 10px;
+  text-decoration: none;
+  color: #fff;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+  vertical-align: top;
+}
+
+
+nav {
+  float: right;
+  vertical-align: top;
+}
+
+nav a:hover {
+  color: yellow;
+}
+
+#active {
+  background-color: grey;
+}
+
+.row {
+  padding-left: 45px;
+}
+
+.column {
+  width: 25%;
```

Comment:
While I understand the desire to use % width here, if you're going to have a fixed-width container (which you do, at 960px wide) as well as fixed-pixel margins/gutter, it's best to do the maths. That way you can avoid stuff like the `padding-left: 45px;` on `.row` (above) and then the `margin-left: -45px;` on the `.column:first-child` (below). Negative margins should only be used as a last resort!

Let's suppose you want three 20px gutters and four columns, because the maths work out better:

(960 - (3 * 20))/ 4 = 225

So now you tweak the styles to be:

```
.column {
  width: 225px;
  margin-right: 20px;
  ...etc...
}
```

... and get rid of that `row` class entirely, while keeping the below `.column:last-child` nullification of the `margin-right`.